admin-dev/themes/new-theme/js/pages/translation-settings/FormFieldToggle.js

```diff
@@ -89,12 +87,9 @@ export default class FormFieldToggle {
         break;
 
       case themes:
-        if ($noThemeOption.is(':selected')) {
-          $themesSelect.val($firstThemeOption.val());
-        }
-
-        this.hide($modulesFormGroup, $emailFormGroup, $noThemeOption);
+        this.hide($modulesFormGroup, $emailFormGroup);
         this.show($themesFormGroup);
+        this.hide($defaultThemeOption);
```

Comment:
```suggestion
```

Reply:
? why?
ThemesOptions is shown and defaultTheme hidden